blocks/carousel/carousel.js

```diff
@@ -1,4 +1,115 @@
+import { readBlockConfig } from '../../scripts/lib-franklin.js';
+import { fetchGraphQL } from '../../scripts/scripts.js';
+
 export default function decorate(block) {
+  if(block.classList.contains('products')) {
+
+    const config = readBlockConfig(block);
+    const query = `
+    {
+      products(filter: { sku: { in: ["6Seats8Sides_760", "8Seats10Sides_760", "5Seats5Sides_760", "10Seats12Sides_760", "7Seats8Sides_760"] } }) {
```

Comment:
Can you take the skus from the block? 

btw, you don't need the first column in the block with the `skus` key,  I removed it from the doc. The first row, first cell just contains the comma separated list of skus.